### ISSUE TITLE
JSON or URL Encoded

### ISSUE THREAD

**Issue by ricardoalcocer** [bradtraversy/php_rest_myblog#4]
https://github.com/bradtraversy/php_rest_myblog/blob/acc9a251b2111eb6952a23223b067a614c1bad03/api/post/create.php#L19

Hey.  I'm playing with this code and for me on my local MAMP environment and using INSOMNIA REST client, I get the data as a query string instead of JSON.  Insomnia is set to POST and send the data as Form URL Encoded.

I have no preference of one or the other, but do you know what's triggering the difference in output formats?

**Comment by ricardoalcocer** [comment]
I'm probably answering my own question here, but is it because you're sending it as JSON in the first place here?             
https://github.com/bradtraversy/php_rest_myblog/blob/acc9a251b2111eb6952a23223b067a614c1bad03/ajaxCall.js#L20

If so, what's the best practice when it comes to grab that data from the request?  Shall I assume JSON?

**Comment by MaxwellEnemuo** [comment]
Yes. When using a REST client simply POST data as JSON because that's what the end point expects.
